client/src/pages/Feed.js

```diff
@@ -586,9 +528,8 @@ const Feed = (props) => {
       default:
         break;
     }
-
     let endpoint = `${baseURL}${objectiveURL()}${filterURL()}${searchURL()}`;
-    postsDispatch({ type: FETCH_POSTS });
+    dispatch(postsActions.fetchPostsBengin());
```

Comment:
```suggestion
    dispatch(postsActions.fetchPostsBegin());
```

Fix function name typo